lessons/pydata/pandas/index.ipynb

```diff
@@ -0,0 +1,2872 @@
+       "</table>\n",
+       "<p>807 rows × 13 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      id         name  height  weight   color      shape  is baby    type 1  \\\n",
+       "0      1    bulbasaur     0.7     6.9   green  quadruped    False     Grass   \n",
+       "1      2      ivysaur     1.0    13.0   green  quadruped    False     Grass   \n",
+       "2      3     venusaur     2.0   100.0   green  quadruped    False     Grass   \n",
+       "3      4   charmander     0.6     8.5     red    upright    False      Fire   \n",
+       "4      5   charmeleon     1.1    19.0     red    upright    False      Fire   \n",
+       "..   ...          ...     ...     ...     ...        ...      ...       ...   \n",
+       "802  803      poipole     0.6     1.8  purple    upright    False    Poison   \n",
+       "803  804    naganadel     3.6   150.0  purple      wings    False    Poison   \n",
+       "804  805    stakataka     5.5   820.0    gray  quadruped    False      Rock   \n",
+       "805  806  blacephalon     1.8    13.0   white   humanoid    False      Fire   \n",
+       "806  807      zeraora     1.5    44.5  yellow   humanoid    False  Electric   \n",
+       "\n",
+       "     type 2  hp  attack  defense  speed  \n",
+       "0    Poison  45      49       49     45  \n",
+       "1    Poison  60      62       63     60  \n",
+       "2    Poison  80      82       83     80  \n",
+       "3       NaN  39      52       43     65  \n",
+       "4       NaN  58      64       58     80  \n",
+       "..      ...  ..     ...      ...    ...  \n",
+       "802     NaN  67      73       67     73  \n",
+       "803  Dragon  73      73       73    121  \n",
+       "804   Steel  61     131      211     13  \n",
+       "805   Ghost  53     127       53    107  \n",
+       "806     NaN  88     112       75    143  \n",
+       "\n",
+       "[807 rows x 13 columns]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Základní zobrazení v notebooku nám ukáže prvních pět a posledních pět řádků v tabulce společně s informací o celkovém počtu řádků a sloupců. Naše tabulka je úmyslně dostatečně malá, aby se s ní pohodlně pracovalo, což ale většinou není ten případ. Proto se podíváme na to, jak si z tabulky vybrat jen to, co nás zajímá."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sloupce a řádky"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sloupce i řádky mají tzv. index, s jehož pomocí se dá k jednotlivým částem tabulky přistoupit. V našem případě má každý řádek automaticky doplněný unikátní číselný index (ten tučně napsaný sloupec nalevo) a jako index pro sloupce nám poslouží prvni řádek CSV souboru s názvy sloupců.\n",
```

Comment:
Já tučně index nemám

Reply:
Nemáš nijak zvýrazněný index? Kde? U sebe nebo při renderu tohoto notebooku? Jak bys to napsal lépe/obecněji?

Reply:
Tak na druhý pokus už je index tučný - sorry, chyba byla buhvíkde ...